src/OpenFund.Infrastructure/Persistence/OpenFundDbContext.cs

```diff
@@ -1,17 +1,7 @@
 ﻿namespace OpenFund.Infrastructure.Persistence;
 
-internal sealed class OpenFundDbContext(DbContextOptions<OpenFundDbContext> options) : DbContext(options)
+public class OpenFundDbContext : IdentityDbContext<User,IdentityRole<Guid>,Guid>
```

Comment:
`public` და `sealed`. ეს არასდროს არ იქნება რამის მშობელი, ამ წამს ყოველ შემთხვევაში. 

ასევე მეტი სილამაზისთვის შეცვალე

`IdentityDbContext<User,IdentityRole<Guid>,Guid>`

TO

`IdentityDbContext<User, IdentityRole<Guid>, Guid>`